release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2779,6 +2779,64 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.49 --pullspecs
+----
+
+[id="ocp-4-15-49-known-issues_{context}"]
+==== Known issues
+
+* IPsec is not supported on Red Hat Enterprise Linux (RHEL) compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exist in each compute node.  (link:https://issues.redhat.com/browse/OCPBUGS-36688 [*OCPBUGS-36688*]) 
+
+[id="ocp-4-15-49-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, an update to the {ibm-cloud-name} Cloud Internet Services (CIS) implementation impacted the upstream Terraform plugin. If you attempted to create an externally-facing cluster on {ibm-cloud-name}, the following error occurred:
++
+[source,terminal]
+----
+ERROR Error: Plugin did not respond
+ERROR
+ERROR with module.cis.ibm_cis_dns_record.kubernetes_api_internal[0],
+ERROR on cis/main.tf line 27, in resource "ibm_cis_dns_record" "kubernetes_api_internal":
+ERROR 27: resource "ibm_cis_dns_record" "kubernetes_api_internal"
+----
++
+With this release, you can use the installation program to create an external cluster on {product-title} without the plugin issue. (link:https://issues.redhat.com/browse/OCPBUGS-54367[*OCPBUGS-54367*]) 
+
+// Awaiting approval from Marco Braga for OCPBUGS-54353
+
+* Previously, when installing a cluster on {aws-first} in existing subnets that were located in edge zones, such as a Local Zone or a Wavelength Zone, the `kubernetes.io/cluster/<InfraID>:shared` tag was missing in the subnet resources of the edge zone. With this release, a fix ensures that all subnets that are used in the `install-config.yaml` configuration file have the required tag. (link:https://issues.redhat.com/browse/OCPBUGS-54353[*OCPBUGS-54353*]) 
+
+// Awaiting approval from Marco Braga for OCPBUGS-53139
+
+* Previously, iSCSI and Fibre Channel devices attached by multipath did not resolve correctly when partitioned. This was caused by improper handling of multipath devices. With this release, a storage bug is fixed so that the partitioned multipath storage is now correctly recognized. (https://issues.redhat.com/browse/OCPBUGS-53139[*OCPBUGS-53139*]) 
+
+* Previously, the *Cluster Settings* page would not properly render during a cluster update if the `ClusterVersion` did not receive a `Completed` update. With this release, the *Cluster Setting* page properly renders even if the `ClusterVersion` has not received a `Completed` update. (link:https://issues.redhat.com/browse/OCPBUGS-53138[*OCPBUGS-53138*]) 
```

Comment:
Is "the `ClusterVersion`" actually "the Cluster Version Operator (CVO)"? (check with an SME but I think this is probably the case)
```suggestion
* Previously, the *Cluster Settings* page would not properly render during a cluster update if the Cluster Version Operator (CVO) did not receive a `Completed` update. With this release, the *Cluster Setting* page properly renders even if the CVO has not received a `Completed` update. (link:https://issues.redhat.com/browse/OCPBUGS-53138[*OCPBUGS-53138*]) 
```